packages/next/src/server/app-render/action-handler.ts

```diff
@@ -630,11 +645,13 @@ export async function handleAction({
       ) {
         // Use react-server-dom-webpack/server.node which supports streaming
         const {
+          createTemporaryReferenceSet,
```

Comment:
I somewhat don't trust using `createTemporaryReferenceSet` from different modules but then we always did the same for `decoreReply`. Ultimately, we always use `renderToReadableStream` from `ComponentMod` not the one from `./react-server.node`. This seems like a module mismatch? Does this just work incidentally? I'm not familiar with `ComponentMod` vs `require(`./react-server.node`)`

Reply:
We don't normally use the node packages anywhere so doesn't make sense to put them on the base entry like we do with the other ones. Really this whole thing needs to be cleaned up but since the sets are the same between node and edge today it'll still function